docs/development/dapr-metrics.md

```diff
@@ -194,3 +194,8 @@ We support only server side metrics.
 * dapr_component_secret_count: The number of operations performed on the secret component
 * dapr_component_secret_latencies: The latency of the response from the secret component
 
+### Job metrics
+
+* dapr_component_job_successCount: The number of job triggers that returned success response
```

Comment:
Thank you for adding to this file!

last nit, can you update the metric to use all underscores and not a mix of underscore and camelcase so its consistent with the other metrics?:
```suggestion
* dapr_component_job_success_count: The number of job triggers that returned success response
```
you'll likely need to update this throughout the files. then assuming CI passes this generally lgtm 👍🏻